### PR TITLE
Update to new x11rb release

### PR DESCRIPTION
**Pull request by psychon**
Hi,

I just noticed your package on https://crates.io/crates/x11rb/reverse_dependencies and thus took a look. I was curious how much breakage the new release caused and was happy that there was not much breakage. Here is the resulting patch to update things.

### CODE REVIEW

**Comment by psychon** [comment]
Oh and from your README I found https://github.com/neXromancers/nixromancers/blob/master/pkgs/tools/misc/uniconize/generic.nix.
```
  nativeBuildInputs = [
    buildPackages.python3
  ];
```
The Python-dependency is gone from x11rb 0.5.0. :-)

File: Cargo.toml [hunk 1]
```diff
@@ -8,6 +8,6 @@ license = "MPL-2.0"
 repository = "https://github.com/neXromancers/uniconize"
 
 [dependencies.x11rb]
-version = "0.4"
+version = "0.5"
+# Disable the "allow-unsafe-code" feature and the libxcb dependency
```
Comment (9ary) [inline Cargo.toml:12]: I guess this does deserve being documented inline and not just in the commit log. :+1:

File: src/main.rs [hunk 2]
```diff
@@ -33,10 +32,8 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     conn.flush()?;
 
     loop {
-        let event = conn.wait_for_event()?;
-        match event.response_type() {
-            CLIENT_MESSAGE_EVENT => {
-                let event: ClientMessageEvent = event.into();
+        match conn.wait_for_event()? {
+            Event::ClientMessage(event) => {
```
Comment (9ary) [inline src/main.rs:36]: Awesome! This was probably my only gripe with the x11rb API, I'm glad you changed it.

**Review by 9ary — APPROVED** [review]
Thanks!

> The Python-dependency is gone from x11rb 0.5.0. :-)

Paging @bb010g on this one. I'll tag a release so we can get this out.

**Pull request merged**